src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,56 @@
+package duke.task;
+
+import java.time.LocalDate;
+
+/**
+ * Represents task of user
+ */
+public class Task {
```

Comment:
You might want to change this class to an abstract class because it doesn't make sense to create a new a Task. It is algo a good practice in OOP because it reduces code repetition.